protocol/standard-validator.md

```diff
@@ -0,0 +1,153 @@
+flow.
+
+### Issues with validations
+
+As can be seen from the many [long `expectedError` strings in the superchain-ops repo](https://github.com/ethereum-optimism/superchain-ops/blob/44f9a09186073456ac1e03c485206b74aa742c30/src/improvements/template/OPCMUpgradeV200.sol#L115-L155),
+there are a variety of errors which have been deemed acceptable for one reason or another.
+
+Before the StandardValidator can be included as a required check in all OPCM operations, we
+need to have confidence that any failures merit reverting the operation.
+
+In order to clarify the cause of these `expectedErrors` it is necessary to clarify the jobs the
+StandardValidator is currently doing:
+
+1. Enforcing the standard configuration.
+2. Checking that values are 'as expected', even for chains which do not satisfy the standard configuration.
+
+To clarify the latter, chains may have a different L1ProxyAdminOwner (ie.
+[Unichain](https://github.com/ethereum-optimism/superchain-registry/blob/0831c2509152b457d865634616925ca6240b219e/superchain/configs/mainnet/unichain.toml#L45)),
+or a different Challenger (ie.
+[Base](https://github.com/ethereum-optimism/superchain-registry/blob/0831c2509152b457d865634616925ca6240b219e/superchain/configs/mainnet/base.toml#L47)).
+In such cases the StandardValidator currently returns an error string (ie. `PROXYA-10` or `PDDG-120`),
+  the current practice is to recognize that this error is expected, but does not allow the StandardValidator
+  to make assertions on the expected value of these roles.
+
+## Proposed Solution
+
+The proposed solution is broken up in to work that can be done prior to upgrade 16, and work
+that will come afterwards.
+
+### Pre-upgrade 16 work
+
+#### 1. OPCM integration:
+
+The `StandardValidator` should:
+- be deployed by [DeployImplementations](https://github.com/ethereum-optimism/optimism/blob/e62c14e64f08ae3cd82973b41315d5797810569b/packages/contracts-bedrock/scripts/deploy/DeployImplementations2.s.sol#L39)
+- included on the OPCM as an [immutable variable](https://github.com/ethereum-optimism/optimism/blob/e62c14e64f08ae3cd82973b41315d5797810569b/packages/contracts-bedrock/src/L1/OPContractsManager.sol#L1785-L1788)
+- listed in the [standard-versions](https://github.com/ethereum-optimism/superchain-registry/blob/0831c2509152b457d865634616925ca6240b219e/validation/standard/standard-versions-mainnet.toml#L1) toml files
+
+#### 2. Improved validations:
+
+The `StandardValidator` should correctly handle certain allowable/expected deviations.
+
+A non-exhaustive list includes:
+
+- `absolutePrestate`: this value varies for each chain. The [current design](https://github.com/ethereum-optimism/optimism/blob/e62c14e64f08ae3cd82973b41315d5797810569b/packages/contracts-bedrock/src/L1/StandardValidator.sol#L61) accepts this value as an input, so no change is needed.
+- Various roles: the default roles should be hardcoded into the StandardValidator, but calls to
```

Comment:
Note that the charter specifically calls out the Base and Uni L1PAO as allowed on their chains only.

Reply:
So that should go into the SV.